web/cobrands/fixmystreet.com/layout.scss

```diff
@@ -83,7 +95,7 @@ h3, h4,
 
 #site-header {
     background: none;
-    border-image-source: url(images/tile-y-border.jpg);
+    border-image-source: url(images/tile-y.jpg);
```

Comment:
Mmhmm. We _could_ use a special, smaller, image (like the old tile-y-border.jpg) to reduce the overall filesize … but we’d only want to do that for high-dpi screens, since standard screens will just reuse the existing tile-y.jpg image. Maybe the extra file and a few more lines in `layout.scss` are worth it, to save retina devices a few kilobytes of download cost?